scripts/data_collector/yahoo/collector.py

```diff
@@ -126,22 +128,32 @@ def _show_logging_func():
 
         interval = "1m" if interval in ["1m", "1min"] else interval
         try:
-            _resp = Ticker(symbol, asynchronous=False).history(interval=interval, start=start, end=end)
+            _resp = Ticker(symbol, asynchronous=False).history(
+                interval=interval, start=start, end=end
+            )
             if isinstance(_resp, pd.DataFrame):
                 return _resp.reset_index()
             elif isinstance(_resp, dict):
                 _temp_data = _resp.get(symbol, {})
                 if isinstance(_temp_data, str) or (
-                    isinstance(_resp, dict) and _temp_data.get("indicators", {}).get("quote", None) is None
+                    isinstance(_resp, dict)
+                    and _temp_data.get("indicators", {}).get("quote", None) is None
                 ):
                     _show_logging_func()
             else:
                 _show_logging_func()
         except Exception as e:
-            logger.warning(f"{error_msg}:{e}")
+            logger.warning(
+                f"{error_msg}:{e}"
+                + "you can Switch to a foreign network or use another data source"
```

Comment:
```suggestion
                + "Your data request fails. This may be caused by your firewall (e.g. GFW). Please switch your network if you want to access Yahoo! data"
```